shared/engine/transport-shared.js

```diff
@@ -159,17 +161,11 @@ class TransportShared extends RobustTransport {
     }
   }
 
-  invoke(arg: Object, cb: any) {
-    // args needs to be wrapped as an array for some reason so let's just do that here
-    const wrappedArgs = {
-      ...arg,
-      args: [arg.args || {}],
-    }
-
+  invoke(arg: InvokeArgs, cb: any) {
     const wrappedInvoke = _wrap({
       enforceOnlyOnce: true,
-      extra: arg.args,
-      handler: args => {
+      extra: arg.args[0],
+      handler: (args: InvokeArgs) => {
```

Comment:
Seems like RobustTransport is typed as any, probably because framed-msgpack-rpc isn't typed, so we need to type args here